### ISSUE TITLE
yarn tsc failing on newly generated app 

### ISSUE THREAD

**Issue by bresmith-wayfair** [spotify/backstage#1463]
## Expected Behavior

`backstage-cli create-app` should generate a new instance of the Backstage app

## Current Behavior

`yarn tsc` fails with the following error: 
```Moving to final location:
  moving        test-app ✔ 

 Building the app:
  executing     yarn install ✔ 
  executing     yarn tsc ◠ error Command failed with exit code 2.
yarn run v1.22.4
$ tsc
node_modules/react-router-dom/index.d.ts:4:119 - error TS2305: Module '"../react-router"' has no exported member 'useLocationPending'.

4 import { MemoryRouter, Navigate, Outlet, Route, Router, Routes, useBlocker, useHref, useInRouterContext, useLocation, useLocationPending, useMatch, useNavigate, useOutlet, useParams, useResolvedLocation, useRoutes, createRoutesFromArray, createRoutesFromChildren, generatePath, matchRoutes, matchPath, resolveLocation } from 'react-router';
                                                                                                                        ~~~~~~~~~~~~~~~~~~

node_modules/react-router-dom/index.d.ts:4:184 - error TS2305: Module '"../react-router"' has no exported member 'useResolvedLocation'.

4 import { MemoryRouter, Navigate, Outlet, Route, Router, Routes, useBlocker, useHref, useInRouterContext, useLocation, useLocationPending, useMatch, useNavigate, useOutlet, useParams, useResolvedLocation, useRoutes, createRoutesFromArray, createRoutesFromChildren, generatePath, matchRoutes, matchPath, resolveLocation } from 'react-router';
                                                                                                                                                                                         ~~~~~~~~~~~~~~~~~~~

node_modules/react-router-dom/index.d.ts:4:303 - error TS2305: Module '"../react-router"' has no exported member 'resolveLocation'.

4 import { MemoryRouter, Navigate, Outlet, Route, Router, Routes, useBlocker, useHref, useInRouterContext, useLocation, useLocationPending, useMatch, useNavigate, useOutlet, useParams, useResolvedLocation, useRoutes, createRoutesFromArray, createRoutesFromChildren, generatePath, matchRoutes, matchPath, resolveLocation } from 'react-router';
                                                                                                                                                                                                                                                                                                                ~~~~~~~~~~~~~~~

node_modules/react-router/index.d.ts:3:60 - error TS2305: Module '"../history"' has no exported member 'PartialLocation'.

3 import { Action, Blocker, History, InitialEntry, Location, PartialLocation, Path, State, To } from 'history';
                                                             ~~~~~~~~~~~~~~~


Found 4 errors.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  executing     yarn tsc ✖ 

Could not execute command yarn tsc

It seems that something went wrong when creating the app 🤔
We are going to clean up, and then you can try again.

 Cleanup:
  remove        temporary directory ✔ 

🔥  Failed to create app!```

## Possible Solution

🤷 

## Steps to Reproduce

1. `backstage-cli create-app`
2. enter anything as a name 


## Context

I'm trying to get a newly scaffolded instance of the Backstage app up and running

## Your Environment

- NodeJS Version (v12): v12
- Operating System and Version (e.g. Ubuntu 14.04): CentOS 7.5
- Browser Information: N/A

**Comment by freben** [comment]
i believe this is addressed by #1502.

**Comment by bresmith-wayfair** [comment]
yep!   thanks again!
